docs/API/LorisRESTAPI.md

```diff
@@ -33,8 +33,8 @@ DELETE is not supported on any resource defined in this API.
 # 1.1 Authentication
 
 If a user is logged in to Loris and can be authenticated using the standard session mechanism,
-no further authentication is required. Requests will be evaluated as requests from that user,
-so that standard Loris modules can simply use the API.
+no further authentication is required. Requests will be evaluated as requests from that user
+so that standard Loris modules are able to use the API.
```

Comment:
@johnsaigle   I'm a little amused here -- is it really condescending to use "simply" here?  it describes a more simple access solution for the modules, not an instruction to users which could have negative connotations.